osgp/platform/osgp-adapter-ws-smartmetering/src/main/java/org/opensmartgridplatform/adapter/ws/smartmetering/infra/jms/messageprocessor/DomainResponseMessageProcessor.java

```diff
@@ -135,7 +134,10 @@ public void processMessage(final ObjectMessage message) {
                             correlationUid, String.valueOf(notificationType)));
 
         } catch (final Exception e) {
-            this.handleError(e, correlationUid, organisationIdentification, deviceIdentification, notificationType);
+            // Logging is enough, sending the notification will be done
+            // automatically by the resend notification job
+            LOGGER.warn("Error: {} occurred for notification type: {} with correlationUid: {}", e.getMessage(),
+                    notificationType, correlationUid, e);
```

Comment:
I wonder if the phrasing of the warning message is a bit too severe for what is going on.
Starting with "Error" might trigger someone seeing this in the logs to look into what should be corrected,
As the issue here will fix itself as suggested by the comment in the code, a log line starting with something like "Unable to deliver notification for ..." could be nicer.
As you have looked more into the application logs than I have, I'll leave it up to your judgement whether you think this could be improved or whether it is fine as it is.

Reply:
Agree. I updated the message.